client/app/telecom/telephony/line/management/terminate/telecom-telephony-line-management-terminate.html

```diff
@@ -33,7 +33,10 @@
                             <i class="ovh-font ovh-font-home ovh-font-1x8 pt-3" aria-hidden="true"></i>
                             <div class="pl-5">
                                 <h4 data-translate="telephony_group_line_terminate_back_address"></h4>
-                                <span translate="telephony_group_line_terminate_address"></span>
+                                <span class="d-block" data-translate="telephony_group_line_terminate_address_name">OVH SAS</span>
```

Comment:
For that purpose you can use the `<address>` element I guess.

Reply:
Please pay attention that you have to refer to the translation key only.

Reply:
`<address>` element will again need `<br>` for line breaks